examples/configs/config_url.json

```diff
@@ -0,0 +1,215 @@
+{
+    "0": {
```

Comment:
Is it a custom format or a file you got from somewhere?
I can't see why `{"0": a, "1": b}` wouldn't be [a, b]

Reply:
excellent point! Keys are useless :)